test/libyul/ObjectCompilerTest.cpp

```diff
@@ -53,7 +53,7 @@ TestCase::TestResult ObjectCompilerTest::run(ostream& _stream, string const& _li
 	AssemblyStack stack(
 		EVMVersion(),
 		m_wasm ? AssemblyStack::Language::Ewasm : AssemblyStack::Language::StrictAssembly,
-		m_optimize ? OptimiserSettings::full() : OptimiserSettings::minimal()
+		m_optimize ? OptimiserSettings::full() : OptimiserSettings::none()
```

Comment:
@chriseth you meant changing this option here or changing a specific test?

Reply:
I think we should not use different settings than in the regular compiler run. Because of that, I think we have to update the tests.

Reply:
There are 2 tests that would require rewrite/update if we want to keep `minimal()` here:
- `test/libyul/objectCompiler/function_series.yul`
- `test/libyul/objectCompiler/jump_tags.yul`

They're pretty short but they basically test jumps which just get optimized out so I'm not sure how to rewrite them.

Reply:
If we add a setting to disable the optimizer, we can disable it for the individual tests.

Reply:
OK. I made the optimization level configurable, which allows us to do `optimizationLevel: none` to disable the optimizer completely: #11319. I set the level to `none` for the two tests mentioned above.

Is that an acceptable solution?